nucypher/cli/painting.py

```diff
@@ -43,14 +43,25 @@ def paint_new_installation_help(new_configuration, federated_only: bool = False,
     emitter(message="Generated keyring {}".format(new_configuration.keyring_dir), color='green')
     emitter(message="Saved configuration file {}".format(new_configuration.config_file_location), color='green')
 
-    if character_name == 'ursula' and not federated_only:
+    # Felix
+    if character_name == 'felix':
+        suggested_db_command = 'nucypher felix createdb'
+        how_to_proceed_message = f'\nTo initialize a new faucet database run:'
+        emitter(message=how_to_proceed_message, color='green')
+        emitter(message=f'\n\'{suggested_db_command}\'', color='green')
+
+    # Ursula
+    elif character_name == 'ursula' and not federated_only:
         suggested_staking_command = f'nucypher ursula stake'
         how_to_stake_message = f"\nTo initialize a NU stake, run '{suggested_staking_command}' or"
         emitter(message=how_to_stake_message, color='green')
 
-    # Give the use a suggestion as to what to do next...
+    # Everyone: Give the use a suggestion as to what to do next
+    vowles = ('a', 'e', 'i', 'o', 'u')
+    character_name_starts_with_vowel = character_name[0].lower() in vowles
```

Comment:
haha, really went for it

Reply:
Important to know if the Character names starts with a

https://upload.wikimedia.org/wikipedia/commons/6/60/Bank_vole.jpg